tests/llmobs/test_experiments.py

```diff
@@ -1491,13 +1505,19 @@ def test_experiment_run(llmobs, test_dataset_one_record):
             },
             "error": {"message": None, "type": None, "stack": None},
         }
-        exp = llmobs.experiment("test_experiment", dummy_task, test_dataset_one_record, [dummy_evaluator])
-        exp._tags = {"ddtrace.version": "1.2.3"}  # FIXME: this is a hack to set the tags for the experiment
-        exp_results = exp.run()
-
-    assert len(exp_results["summary_evaluations"]) == 0
-    assert len(exp_results["rows"]) == 1
-    exp_result = exp_results["rows"][0]
+        with mock.patch("ddtrace.llmobs._experiment._ExperimentRunInfo") as mock_experiment_run_info:
+            # this is to ensure that the UUID for the run is always the same
+            mock_experiment_run_info.return_value = run_info_with_stable_id(0)
+            exp = llmobs.experiment("test_experiment", dummy_task, test_dataset_one_record, [dummy_evaluator])
+            exp._tags = {"ddtrace.version": "1.2.3"}  # FIXME: this is a hack to set the tags for the experiment
```

Comment:
why do we need this hack? or is this comment just here to note that we need to figure out a better way to set tags on experiments?

Reply:
i ended up removing those actually